src/core/mainwindow.cpp

```diff
@@ -1726,6 +1739,21 @@ void MainWindow::UpdateTrackSliderPosition() {
 
 }
 
+void MainWindow::UpdateTaskbarProgress(const bool visible, const double position, const double length) {
+
+  QVariantMap map;
+  QDBusMessage msg = QDBusMessage::createSignal(QStringLiteral("/org/strawberrymusicplayer/strawberry"),
+		                                QStringLiteral("com.canonical.Unity.LauncherEntry"), 
```

Comment:
Please avoid the use of tabs instead of spaces. Here is also a trailing space at the end of the line.
But I prefer to keep this on one line.

Reply:
Done